roles/sft-server/tasks/traffic.yml

```diff
@@ -30,7 +31,6 @@
     to_port: '22'
     state: reloaded
 
-
```

Comment:
Oh, it looked stray, so I removed it. What does it mean? Is it some convention or could we make it more obvious?

Reply:
No convention there. Just me thinking that it would be a good idea to visually separate the firewall rules for media (UDP) from the rest - above TCP, and below the reverse-proxy / vhost tasks. 

If you felt the urge of removing then. sure. I was just curious.